src/Svg/String.elm

```diff
@@ -1,26 +1,29 @@
 module Svg.String exposing
-    ( Attribute
-    , Html
-    , Svg
-    , circle
-    , defs
-    , feColorMatrix
-    , feGaussianBlur
-    , feOffset
-    , filter
-    , g
-    , map
-    , node
-    , rect
-    , svg
-    , text
-    , text_
-    , toHtml
-    , toString
-    , tspan
-    , use
+    ( toHtml, toString
+    , Svg, Html, Attribute, text, node, map
```

Comment:
is this change made by elm-format?

Reply:
yes, the order that elm-format uses here is the same order of the elements inside the documentation comments in the file. I found that out when changing the @doc lines in the comments.